cell_rebalance.py

```diff
@@ -0,0 +1,44 @@
+#this is not meant to be in the main code
+#each cell has the same i2c fuel gauge and therefore
+#the address needs to be changed so we can communicate with
+#each cell simultaneously (it's a programmable address IC)
+#this only needs to run once ever and the addresses are stored in EEPROM
+#create a main.py, create object, and then run rebalance()
+
+import GPIO
+import i2c
+import BMS
+import time
+
+class Cell:
```

Comment:
Even though this won't be in the main code, it's better not not have the duplicate `Cell` class name. Maybe `CellRebalancer`?